data/games/under_night_in_birth_exe_late_st/movespecs/akatsuki.json

```diff
@@ -0,0 +1,142 @@
+{
+  "name": "Akatsuki",
+  "movelists": [
+    [
+      "5A",
+      "2A",
+      "c.B",
+      "f.B",
+      "2B",
+      "c.C",
+      "f.C",
+      "2C",
+      "j.A",
+      "j.7/9B",
```

Comment:
Do we want to split these out into two different inputs or nah? I know we don't really have slashes anywhere else.

Reply:
I was wondering about that too. I figured they were different inputs for the same move, so I'd keep it as a slash? I don't feel strongly about that though so if you think it should be two inputs that's fine by me.

Reply:
I changed my mind about this. / doesn't show up in numpad notation as far as I can tell, so I think we should list them as separate inputs, even if they're the same move.

Reply:
Okay cool. We can always revisit later if necessary.